mailpoet/lib/Subscribers/SubscriberSubscribeController.php

```diff
@@ -153,6 +160,22 @@ public function subscribe(array $data): array {
     return $meta;
   }
 
+  /**
+   * Checks if the subscriber is subscribed to any segments in the form
+   *
+   * @param  FormEntity       $form       The form entity
+   * @param  SubscriberEntity $subscriber The subscriber entity
+   * @return bool True if the subscriber is subscribed to any of the segments in the form
+   */
+  public function isSubscribedToAnyFormSegments(FormEntity $form, SubscriberEntity $subscriber): bool {
```

Comment:
I couldn't find an existing function that would check if a subscriber is already subscribed to the segments of a form.
Let me know if there is one and I missed it!

Reply:
I also could not find such a method. Maybe we had something in the past, but after the new form editor was launched, it's gone. Thanks for adding it!